conda/conda_build_config.yaml

```diff
@@ -7,10 +7,10 @@ scalar:
   - complex
 
 python:
-  - 3.8
   - 3.9
   - 3.10
   - 3.11
+  - 3.12
```

Comment:
Last time I tested python 3.12 with the conda build of funtofem it didn't work.

Reply:
I also ran into issues building against 3.12 when it first came out. It tends to take a while for all of our dependencies to upgrade their conda packages for new Python versions.

That said, a 3.12 build for funtofem will definitely fail at the moment since TACS doesn't have any Python 3.12 conda packages (and tacs is listed as a dependency for funtofem in the conda recipe). I tested out building a conda package for tacs on 3.12 this morning and seemed to have success. So 3.12 support should be available on the tacs side with the new tacs version [bump](https://github.com/smdogroup/tacs/pull/299) this week.